### PR TITLE
ci: push test results to elasticsearch instance

### PR DESCRIPTION
**Pull request by nashif**
Move away from opensearch and switch to elasticsearch for reporting.
The results will be available at https://kibana.zephyrproject.io/

Signed-off-by: Anas Nashif <anas.nashif@intel.com>

### CODE REVIEW

File: .github/workflows/twister.yaml [hunk 1]
```diff
@@ -293,9 +293,9 @@ jobs:
         run: |
           pip3 install opensearch-py
```
Comment (stephanosio) [inline .github/workflows/twister.yaml:294]:
```suggestion
          pip3 install elasticsearch
```

**Review by stephanosio — CHANGES_REQUESTED** [review]

**Review by stephanosio — COMMENTED** [review]
There is a pylint failure.

**Review by stephanosio — APPROVED** [review]

**Comment by stephanosio** [comment]
Will hold off merging this until the RC1 is tagged later today in order to prevent any potential CI breakages at the last minute before the feature freeze.

**Comment by nashif** [comment]
> Will hold off merging this until the RC1 is tagged later today in order to prevent any potential CI breakages at the last minute before the feature freeze.

why would this break? and why it is ok to break after rc and not before it, if something breaks, it can be fixed quickly. That is a no brainer and simple enough. 
I am interested in starting the collection of data before the release cycle starts

**Comment by stephanosio** [comment]
> why would this break?

Just trying to minimise risk.

> and why it is ok to break after rc and not before it, if something breaks, it can be fixed quickly. That is a no brainer and simple enough.

We are only hours away from the freeze and we would not want feature PRs not making this release because of failing CI, especially given that a rebase results in the dismissal of approvals.

**Comment by nashif** [comment]
> We are only hours away from the freeze and we would not want feature PRs not making this release because of failing CI, especially given that a rebase results in the dismissal of approvals.

seriously now, you are basically introducing an RC0 here, risk management comes later in the release cycle, not when the development is still going on.

**Comment by stephanosio** [comment]
Merging this now since this step runs on push (and not on pull requests), so the impact on pull requests CI runs should be none.

**Pull request merged**